DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/compute_units/src/build.ninja

```diff
@@ -0,0 +1,30 @@
+source_file = compute_units.cpp
```

Comment:
Actually in the documentation on compiling above, you use explicit `make` instead of `cmake --build`, so I cannot see how some Ninja code is getting involved while not using Ninja...

Reply:
All FPGA samples are temporarily using ninja for Windows build while we await the resolution of a DPC++ Windows compatibility issue with CMake.

Reply:
Yes, I understood this by reviewing another example...